app/javascript/packages/document-capture/components/suspense-error-boundary.tsx

```diff
@@ -0,0 +1,63 @@
+import { Component, Suspense } from 'react';
+import type { ReactNode } from 'react';
+import { RetrySubmissionError } from './submission-complete';
```

Comment:
Should this be `import type` for consistency with convention for importing types?

```suggestion
import type { RetrySubmissionError } from './submission-complete';
```